roles/openafs_devel/tasks/build.yaml

```diff
@@ -61,7 +61,7 @@
     --with-afs-sysname={{ afs_devel_sysname | default('amd64_linux26') }}
     --{{ afs_devel_build_client | ternary('enable','disable') }}-kernel-module
     {{ _afs_devel_client_opts if afs_devel_build_client else _afs_devel_server_opts }}
-  environment: "{{ afs_devel_client_env if afs_devel_build_client else _afs_devel_server_env }}"
+  environment: "{{ _afs_devel_client_env if afs_devel_build_client else _afs_devel_server_env }}"
```

Comment:
Btw, this afs_devel_client_env is supposed to be the variable you set in your inventory to set a custom value. _afs_devel_etc is the values set in the vars directory which are constant per os type and are the fall backs.

Same goes for line 63 above this.

Reply:
Yes - the afs_devel_client_opts and afs_devel_client_env are set in the inventory.  The previous commit(s) were fixing how ansible handles the precedence of setting variables (see: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable).  Vars set via include_vars take precedence over those set in the inventory.

Reply:
Yes, vars set in include_vars take precedence over the inventory, that's why those were prefixed with underscores, to be a kind of fall back.  I guess something changed over time to break the convention, sorry about that. It will be sorted out when we integrate the new custom modules. Thanks.